src/ext/response-targets.js

```diff
@@ -83,7 +83,7 @@
                 evt.detail.xhr.status !== 200) {
                 if (evt.detail.target) {
                     if (htmx.config.responseTargetPrefersExisting) {
-                        evt.detail.shoudSwap = true;
+                        evt.detail.shouldSwap = true;
```

Comment:
This is actually a change to the extension, and seems to be a bug, so it should be in its own PR with a test to catch whatever isn't being caught here.

Reply:
Everything else looks good

Reply:
Okay.
Could you open an issue for this bug?

Reply:
If you don't, I probably will, but since you're the one who seems to want it done, you'll definitely get to it sooner than I will ;)

Reply:
I am not a htmx user. This is theoretical PR.

Reply:
Reverted.